### DISCUSSION TITLE
No detections in very large gigapixel images

### DISCUSSION THREAD

**Discussion by Nisse123** [obss/sahi#815] [General]
Is there some kind of limitation or special handling of detections in very large gigapixel images?
For a specific image I got no detections using get_sliced_predictions with Yolov5. However, if a run the detection on the individual tiles with the same tile size used in get_sliced_predictions,  I get a lot of good detections. So I tried to divide the large image in half, and then I begin to get detections in the individual image parts but some of the detections are false (seems to be rather large random objects). Then I divided the image in four parts and then almost all detections are good. I tried all combinations of postprocess_type, postprocess_match_metric and perform_standard_pred, but I did not see any improvements on the very large image. I am using the models native resolution as tile_size with 0.1 overlap. I also tried with no overlap but no improvement.

**Comment by fcakyon** [comment]
Please try with slice_size=640.

**Comment by yanqianshilu** [comment]
Have you solved your problem? I have the same problem as you. The original size of my data set is 5472*3648, and the AP50 detected is only 15. When I reduce the image size to 2800*2800, the AP50 is more than 40. When I further reduce the image size, the accuracy will be further improved.

**Reply by Nisse123** [reply]
Not really. 
It would be nice with some kind of rule of thumb here. Like for a model with native resolution of X*Y, for the largest and smallest object expected to be detected, select this size of slice and this overlap ration and also the other parameters.
Reducing the resolution means that small objects eventually will not be detected.
